### DISCUSSION TITLE
Contributors File Question

### DISCUSSION THREAD

**Discussion by Elliria** [autokey/autokey#805] [Maintenance goals]
In the **Testing** section of the **Contributors.rst** file in the [develop branch](https://github.com/autokey/autokey/blob/develop/CONTRIBUTORS.rst#testing) and in the [master branch](https://github.com/autokey/autokey/blob/master/CONTRIBUTORS.rst#testing), it says:

> The current (0.95.x) master tests are deprecated and won’t work. They are still on Python 2 and nobody cared to update them throughout the years, not even the original developer. They test things long gone in the project code… The old tests are scheduled to be replaced by a new suite in the develop branch, which will be merged as 0.96.0.
> 
> The current situation is a bit unfortunate: The new suite lives in the develop branch, which accumulates new features, but which can't be backported to master without introducing merge conflicts later on. So until develop gets merged in, you’d have to switch to develop to run the tests.
> 

Did that merge get done and should those paragraphs be deleted or is there still something we need to do here?

**Comment by josephj11** [comment]
@BlueDrink9 will have to answer this one, but he did say something recently about the tests being targeted at develop.

You've been mucking about in the test files, etc. for the Python version upgrades. If you know where they are, you could do a straight diff between the files in each branch to see for yourself. You could start by looking at the respective file modification dates (if you haven't already changed them).

If you want to do it the "right" [way](https://devconnected.com/how-to-compare-two-git-branches/), you can run a `git diff`, but to do that you'll have to figure out how to specify where they are (URLs?) because they're probably not both local to your machine...
And, you don't really want to see all the differences between the two branches, just the ones you're interested in.

**Comment by Elliria** [comment]
Thank you for the **git diff** suggestion. I'll see what I can learn about using that to figure this out. Meanwhile, I happened to come across a related issue that I should have searched for before starting this discussion. It's issue #718.
